tests/e2e/simple_rp_roa_test.py

```diff
@@ -77,7 +78,58 @@ def wait_until_ca_has_resources(ca_handle, asn, v4, v6):
         ca = krill_ca_api.get_ca(ca_handle)
         f = attrgetter("resources")
         res = f(ca)
-        return res.asn == asn and res.v4 == v4 and res.v6 == v6
+        return set(res.asn) == set(asn) and set(res.v4) == set(v4) and set(res.v6) == set(v6)
+
+    #
+    # define some helper functions
+    #
+    def add_ca(ca_handle):
+        logging.info(f'-> Adding CA "{ca_handle}"')
+        krill_ca_api.add_ca(krill_ca_api_lib.AddCARequest(ca_handle))
+
+        # rfc8183_request = krill_ca_api.get_ca_publisher_request("ta", format="json")
+        # logging.info(f'XIMON: req={rfc8183_request}')
+        krill_ca_api.update_ca_repository(
+            ca_handle,
+            rfc8183='embedded')
```

Comment:
Note that I made changes in this PR to support NOT using embedded mode (e.g. making Krill use our custom root CA certificate so that it can trust the connection out via NGINX back to itself) but hit some final problem and reverted to using "embedded" when I realized that actually still works. I should switch this back to non-embedded and find out what didn't work and why.